app/src/main/res/layout/fragment_session_detail.xml

```diff
@@ -155,14 +164,91 @@
                     android:gravity="top"
                     android:text="@{session.desc}"
                     android:textAppearance="@style/TextAppearance.App.Body1"
-                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintBottom_toTopOf="@id/detail_sessions_indicator_background"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
                     app:layout_constraintTop_toBottomOf="@id/session_room"
                     app:layout_constraintVertical_bias="0.0"
                     tools:text="A material metaphor is the unifying theory of a rationalized space and a system of motion. The material is grounded in tactile reality, inspired by the study of paper and ink, yet technologically advanced and open to imagination and magic"
                     />
 
+                <View
+                    android:id="@+id/detail_sessions_indicator_background"
+                    android:layout_width="match_parent"
```

Comment:
`match_parent` is not recommended in ConstraintLayout.
`0dp` is correct one in this case.

Reply:
Oh, that's right! I will fix it too 🙌